src/acrquery/azext_acrquery/_help.py

```diff
@@ -0,0 +1,28 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['acr query'] = """
+    type: group
+    short-summary: KQL querying against an ACR.
+"""
+
+helps['acr query'] = """
+    type: command
+    short-summary: KQL querying against an ACR.
```

Comment:
We can be more explicit in the summary. KQL is not something that everyone understands as an acronym. We should spell out. Also, we need to explain what are the searchable terms - one would assume that I can search everything.

Do we have longer summary that we can add?